services/openshift/scripts/openshift_provision

```diff
@@ -129,6 +129,7 @@ if [ ! -f ${ORIGIN_DIR}/configured.templates ]; then
 
     #EAP
     /opt/adb/openshift/templates/eap64-mysql-persistent-s2i.json
+    /opt/adb/openshift/templates/eap64-basic-s2i.json
```

Comment:
+1 good to see it comes back. Please make sure to get this merged prior code freeze.